inc/connector/namespace.php

```diff
@@ -422,14 +500,41 @@ function schedule_event( $event ) {
 	$job->start = $job->nextrun;
 	$job->args = $event->args;
 
-	$job->save();
+	$result = $job->save();
+	if ( ! $result && $wp_error ) {
+		return new WP_Error(
+			'could_not_set',
```

Comment:
Not clear what is failing from the error message, could not set what? maybe `cron_schedule_save_failed` would be clearer?

Reply:
I just copied this from core’s `_set_cron_array` function - there are new core tests that check the WP_Error responses so I started with getting those to pass. The text message should make it clearer but I’ll revisit if this needs to match or not since I excluded some more pre flight filter tests

Reply:
I'm going to keep this error message and code as-is to match WP core I think. I've updated one other to be specific to cavalcade, the `invalid_event` one.